cerbero/build/cookbook.py

```diff
@@ -390,6 +394,12 @@ def _load_recipes(self):
                     st.touch()
                 else:
                     self.reset_recipe_status(recipe.name)
+            # check that we use the same config md5
+            # Use getattr as config_hash was added later
+            saved_config_hash = getattr(st, 'config_hash', 0)
```

Comment:
Honestly I have no idea. I just followed what was there on line 387. I'm just using the same approach as the "file_hash" attribute